python/ray/util/client/common.py

```diff
@@ -84,6 +84,12 @@
 
 CLIENT_SERVER_MAX_THREADS = float(os.getenv("RAY_CLIENT_SERVER_MAX_THREADS", 100))
 
+# Large objects are chunked into 64 MiB messages
+OBJECT_TRANSFER_CHUNK_SIZE = 64 * 2 ** 20
+
+# Warn the user if the object being transferred is larger than 2 GiB
+OBJECT_TRANSFER_WARNING_SIZE = 2 * 2 ** 30
+
 
```

Comment:
This should be present after https://github.com/ray-project/ray/pull/22100, correct?

Reply:
Yup!